localisation/000_wc_traits.csv

```diff
@@ -622,3 +622,7 @@ proud_desc;For some reason this character is proud or even selfish. Proud charac
 pneumonic_desc;This character suffers from pneumonia � a deadly infection of the lungs.;;Dieser Charakter leidet an einer Lungenentz�ndung, einer oftmals t�dlichen Infektion.;;;;;;;;;;;x
 lefthanded_desc;This character is left-handed. People who are left-handed tend to have a considerable advantage in combat.;;Dieser Charakter ist Linksh�nder. Personen, die Linksh�nder sind, haben im Kampf oft einen erheblichen Vorteil.;;;;;;;;;;;x
+creature_wickerman;Wickerman;;Weidenmann;;;;;;;;;;;x
+creature_wickerman_desc;This character is a wickerman. Wickermen are created by Throsic covens from inanimate organic materials such as wood, flesh, fur, and bones and are animated by ancient Drust death magic. They are powered by the souls of mortal victims or by the spirits of ancient Drust and thus give the spirits physical form.;;Dieser Charakter ist ein Weidenmann. Weidenmann werden von throsischen Zirkeln aus unbelebten organischen Materialien wie Holz, Fleisch, Fell und Knochen erschaffen und durch die antike Todesmagie der Drust animiert. Sie werden von den Seelen der sterblichen Opfer oder von den Geistern der antiken Drust angetrieben und geben den Geistern somit eine physische Form.;;;;;;;;;;;x
+creature_wicker_golem;Wicker Golem;;Weidengolem;;;;;;;;;;;x
+creature_wicker_golem_desc;This character is a wicker golem. Wicker golems are created by Throsic covens from inanimate organic materials such as wood, flesh, fur, and bones and are animated by ancient Drust death magic. They are powered by the souls of mortal victims or by the spirits of ancient Drust and thus give the spirits physical form. Due to an excruciating soul-ripping process, most victims' souls are shattered to the point of losing all cognitive intelligence, although some of the souls remain aware of just how trapped they are within their wicker bodies.;;Dieser Charakter ist ein Weidengolem. Weidengolems werden von throsischen Zirkeln aus unbelebten organischen Materialien wie Holz, Fleisch, Fell und Knochen erschaffen und durch die antike Todesmagie des Drust animiert. Sie werden von den Seelen der sterblichen Opfer oder von den Geistern der antiken Drust angetrieben und geben den Geistern somit eine physische Form. Durch einen qualvollen, seelenzerst�renden Prozess werden die Seelen der meisten Opfer so weit zerrissen, dass sie ihre ganze geistige Kraft verlieren, obwohl einige der Seelen sich bewusst bleiben, wie gefangen sie in ihren Weidenk�rpern sind.;;;;;;;;;;;x
```

Comment:
I'd write throsic/throsian without capital, since no specific coven is meant.

Reply:
I know literally nothing about the Drust. If Throsic is a culture or nationality. like French or German, it should be capitalized in English, otherwise it can be lowercase. Ultimately though, I think this is a style preference, so pick and choose which for the mod; doesn't matter.

Reply:
Ok, seems like I was wrong here. Capitalised is fine :)